persistence/eclipselink/src/main/java/org/apache/polaris/extension/persistence/impl/eclipselink/PolarisEclipseLinkStore.java

```diff
@@ -294,14 +295,25 @@ List<ModelEntity> lookupFullEntitiesActive(
 
     // Currently check against ENTITIES not joining with ENTITIES_ACTIVE
     String hql =
-        "SELECT m from ModelEntity m where m.catalogId=:catalogId and m.parentId=:parentId and m.typeCode=:typeCode";
+        "SELECT m from ModelEntity m where"
+            + " m.catalogId=:catalogId and m.parentId=:parentId and m.typeCode=:typeCode and m.id > :tokenId";
+
+    long tokenId = -1;
```

Comment:
There used to be a constant for this in `EntityIdPageToken`; putting a magic number here is not a good idea

Reply:
refactored